### DISCUSSION TITLE
当数据量只有一条的时候，折线图上没有点，没有任何交互内容

### DISCUSSION THREAD

**Discussion by lxzy-yun** [antvis/G2#5671] [Q&A]
```
import { Chart } from '@antv/g2';

const chart = new Chart({
  container: 'container',
  autoFit: true,
});

chart
  .line()
  .data([
    {date: '2',close: 23}
  ])
  .encode('x', 'date')
  .encode('y', 'close');

chart.render();
```
当数据量只有一条的时候，折线图上没有点，没有任何交互内容
![image](https://github.com/antvis/G2/assets/26783322/43dc009d-f2b0-41d4-8bef-2a51c55fcdff)

请问是特意设计还是有是bug？
版本号是5.1.5

**Comment by pearmini** [comment]
这个正常的现象，因为：**一个点没有办法绘制一条线**

**Comment by hustcc** [comment]
@lxzy-yun  看看这个 [issue](https://github.com/antvis/G2/issues/5685) 的做法是否可行，有兴趣，也可以来 PR。
